.github/workflows/common_tests.yml

```diff
@@ -11,26 +11,25 @@
 # script to match the changes you made here
 # and rerun the script).
 
-
 name: Common Tests
-'on':
+"on":
   push:
     branches:
-    - master
-    - wip
+      - master
+      - wip
     paths:
-    - common/**
-    - .github/workflows/common_tests.yml
```

Comment:
You shouldn't need to change this file... if you revert these changes,  do you still get errors?